components/InstallHandlerBtn.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <q-btn
+    label="Manage Credentials"
```

Comment:
I don't know that users will know what "Manage Credentials" means -- and if they click this button it also won't "manage their credentials". I think we need some different text here like "Allow Wallet" -- and then we'll need some need in text where the button is used explaining what it's for.

```suggestion
    label="Manage Credentials"
```

Reply:
Maybe we also want to call this an `AllowWalletButton` too.

Reply:
Took the Suggestion and renamed the button to `AllowWalletButton` https://github.com/digitalbazaar/bedrock-vue-wallet/pull/29/commits/b0524dc2ea59ec379cc4dfcefa8fb30c6cd96c4f